### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 30

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#206] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152254449-1515642f-8a8c-4579-b294-598ae1b4f0d1.png)

**Comment by LuginiAndrea** [comment]
S = BEPPESIGNORI = {B: 1, E: 2, P: 2, S: 1, I: 2, G: 1, N: 1, O: 1, R: 1}

1. $$\frac{12!}{2! \cdot 2! \cdot 2!}$$
2. Per complemento: $$\frac{12!}{2!^3} - \frac{11!}{2!^3}$$
3. ab $=$ BE. Notiamo subito come i due insiemi sono disgiunti (essendoci una sola B, questa non può esserei _sia_ all'inizio _sia_ alla fine). Risulta quindi $$\frac{10!}{2! \cdot 2!} \cdot 2 - 0$$
